dsgrid/dataset/dataset_schema_handler_base.py

```diff
@@ -11,6 +11,7 @@
 from dsgrid.exceptions import DSGInvalidDataset, DSGInvalidQuery
 from dsgrid.dimension.time import TimeDimensionType
 from dsgrid.query.query_context import QueryContext
+from dsgrid.units.energy import convert_units
```

Comment:
How does this change when we add other sorts of metrics (e.g., capacity, population/counts)?

Reply:
I would probably use 
```
import dsgrid.units.energy as energy
energy.convert_units()
```
I can make the change now.